crates/e2e/src/setup/services.rs

```diff
@@ -670,3 +698,32 @@ pub async fn clear_database() {
 }
 
 pub type Db = sqlx::Pool<sqlx::Postgres>;
+
+/// Clap does not allow you to overwrite CLI arguments easily. This
+/// function loops over all provided arguments and only keeps the
+/// last one if there are multiples.
+fn ignore_overwritten_cli_params(mut params: Vec<String>) -> Vec<String> {
```

Comment:
As this is very generic, I would move it to the `setup/mod.rs` file so it can be easily reused in other e2e tests in the future.